.npmignore

```diff
@@ -0,0 +1,3 @@
+lib_covered
+coverage
+
```

Comment:
we should probably add *tests-xunit.xml* here as well as to the `.gitignore` and untrack it.